docs/windows.md

```diff
@@ -295,7 +296,13 @@ curl.exe -LO https://raw.githubusercontent.com/antrea-io/antrea/main/hack/window
 .\Install-OVS.ps1 -ImportCertificate $false -Local -LocalFile <PathToOVSPackage> # Production
 ```
 
-Verify the OVS services are installed.
+If you want to containerise OVS for containerd runtime , use the ContaineriseOVS parameter as true.
+
+```powershell
+.\Install-OVS.ps1 -ContaineriseOVS $true 
+```
+
+Verify the OVS services are installed.(not applicale if you containerise OVS)
```

Comment:
Add space after the period.

Reply:
Typo for "applicale". BTW, maybe this line can be removed?

Reply:
@wenyingd You mean we should remove the entire section where we are verifying that ovs-services are installed or not using get-service command?

Reply:
No, I mean it looks wierd with the changes. The verification is for OVS services, but the added command actually does not add OVS services. Is it better to add the example command with `-ContaineriseOVS $true` after the verification part?

Reply:
Okay , understood